can/io/asc.py

```diff
@@ -55,10 +55,14 @@ def __iter__(self):
             temp = line.strip()
             if not temp or not temp[0].isdigit():
                 continue
+            is_fd = False
             try:
                 timestamp, channel, dummy = temp.split(
                     None, 2
                 )  # , frameType, dlc, frameData
+                if channel == "CANFD":
+                    timestamp, _, channel, dir, dummy = temp.split(None, 4)
```

Comment:
`dir` is a Python built-in name. Please use another variable name.

Reply:
Done